modules/nf-core/backsub/main.nf

```diff
@@ -1,28 +1,37 @@
 process BACKSUB {
     tag "$meta.id"
     label 'process_single'
+    tag "modules"
+    tag "modules_nfcore"
+    tag "backsub"
```

Comment:
you don't need tags here...only in the test file

Reply:
Thanks!